cmd/zpool/zpool_vdev.c

```diff
@@ -721,8 +722,16 @@ make_leaf_vdev(nvlist_t *props, const char *arg, uint64_t is_log)
 		char *value = NULL;
 
 		if (nvlist_lookup_string(props,
-		    zpool_prop_to_name(ZPOOL_PROP_ASHIFT), &value) == 0)
+		    zpool_prop_to_name(ZPOOL_PROP_ASHIFT), &value) == 0) {
 			zfs_nicestrtonum(NULL, value, &ashift);
+			if (ashift != 0 &&
+			    (ashift < ASHIFT_MIN || ashift > ASHIFT_MAX)) {
+				(void) fprintf(stderr, dgettext(TEXT_DOMAIN,
```

Comment:
Since this is in the command and not a library you'll want to use `gettext()` for consistency with the errors above.  The text domain is already set to TEXT_DOMAIN.